spring-boot-docs/src/main/asciidoc/appendix-application-properties.adoc

```diff
@@ -194,6 +194,7 @@ content into your application; rather pick only the properties that you need.
 	server.ssl.trust-store-provider= # Provider for the trust store.
 	server.ssl.trust-store-type= # Type of the trust store.
 	server.tomcat.accept-count= # Maximum queue length for incoming connection requests when all possible request processing threads are in use.
+	server.tomcat.accesslog.fileDateFormat=yyyy-MM-dd # Customized date format in the access log file name.
```

Comment:
Keys are ordered alphabetically and use a canonical name (`file-data-format`). The description must match the one in the field's Javadoc. Again, check other keys for consistency.